src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/AngularDistribution/Tabulated/Builder/src/equalSize.hpp

```diff
@@ -0,0 +1,13 @@
+template< typename T >
+void equalSize( T& t, std::string name ){
```

Comment:
Don't you have a function lying around taking multiple ranges and then verifying they're all the same size?

Reply:
Yeah. This function was created before I had that function.